### PR TITLE
Stm32mp15 eval fixes

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr.bb [hunk 1]
```diff
@@ -30,6 +30,15 @@ do_deploy() {
     install -m 0644 uEnv.txt ${DEPLOYDIR}
 }
 
+do_install() {
+    install -m 0644 boot.scr ${D}/boot.scr
+    install -m 0644 uEnv.txt ${D}/uEnv.txt
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr.bb:35]: do we need to have it under root? or is the u-boot script (distro) generic enough to look for content under /boot?

Comment (igoropaniuk) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr.bb:35]: AFAIK, distroboot expect boot.scr to be in root of filesystem

Comment (igoropaniuk) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr.bb:35]: but yeah, we can move uEnv to `boot/` dir

Comment (igoropaniuk) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr.bb]: done

**Review by quaresmajose — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**